modules/system/models/MailLayout.php

```diff
@@ -41,6 +41,13 @@ class MailLayout extends Model
         'content_html'          => 'required',
     ];
 
+    /**
+     * @var array Options array
+     */
+    protected $casts = [
```

Comment:
Please use the `jsonable` property instead: http://octobercms.com/docs/database/model#property-jsonable